src/state/airdrop/selectors.ts

```diff
@@ -1,10 +1,29 @@
+import { differenceBy } from 'lodash';
 import { createSelector } from 'reselect';
 
+import { Wallet } from 'app/consts';
+import { HDSegwitP2SHAirWallet } from 'app/legacy';
 import { ApplicationState } from 'app/state';
 
+import { WalletsState } from '../wallets/reducer';
 import { AirdropState } from './reducer';
 
 const local = (state: ApplicationState): AirdropState => state.airdrop;
+const walletsState = (state: ApplicationState): WalletsState => state.wallets;
 
 export const thankYouSeen = createSelector(local, state => state.thankYouSeen);
 export const thankYouFlowCompleted = createSelector(local, state => state.thankYouFlowCompleted);
+export const airdropReadyWallets = createSelector(walletsState, state =>
+  state.wallets.filter((w: Wallet) => w.type === HDSegwitP2SHAirWallet.type),
+);
+
+export const subscribedWallets = createSelector([walletsState, local], (walletsState, airdropState) =>
```

Comment:
create a selector for subcribedIds and the 
`createSelector(walletsSelectors.wallets, subcribedIds, (wallets, subIds) => ...)`

Reply:
done